tests/common_setup.py

```diff
@@ -36,69 +43,85 @@ def wait_for_containers(expected_containers, defined_in):
 
 @pytest.fixture(scope="function")
 def standard_setup_one_client(request):
-    restart_docker_compose()
-    reset_mender_api()
-
+    env = container_factory.getStandardSetup(1)
+    env.setup()
 
-def setup_set_client_number_bootstrapped(clients):
-    docker_compose_cmd("scale mender-client=%d" % clients)
     ssh_is_opened()
+    reset_mender_api()
 
-    auth.reset_auth_token()
-    auth_v2.accept_devices(clients)
+    def fin():
+        env.teardown()
 
+    request.addfinalizer(fin)
```

Comment:
Here you can write it as: `request.addfinalizer(env.teardown)`

Reply:
Nicely spotted! Thanks. I will add it in all fixtures.